### ISSUE TITLE
Pretrained resnet for HPE

### ISSUE THREAD

**Issue by sebo313** [tensorflow/models#6153]
### System information
Linux Ubuntu 16.04, Tensorflow 1.18 from binary, 
GPU GeForce GTX 1070

###Questions:

Which available pretrained tensorflow resnet has the highest accuracy in human pose estimation?

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
What is the top-level directory of the model you are using
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by sebo313** [comment]
What is the top-level directory of the model you are using: N/A
Have I written custom code: No
OS Platform and Distribution: Linux Ubuntu 16.04,
TensorFlow installed from: Binary
TensorFlow version: 1.18
Bazel version: N/A
CUDA/cuDNN version: N/A
GPU model and memory: GeForce GTX 1070
Exact command to reproduce: N/A

###Question:

Which available pretrained tensorflow resnet has the highest accuracy in human pose estimation?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be an old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
